easybuild/easyconfigs/i/IntelPython/IntelPython-2.7.15-2019.2.066.eb

```diff
@@ -0,0 +1,43 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'IntelPython'
+version = '2.7.15'
+pysver = version[0]
+relver = '2019.2.066'
+versionsuffix = '-%s' % relver
+
+homepage = 'https://software.intel.com/en-us/intel-distribution-for-python'
+
+description = """
+ Intel® Distribution for Python. Powered by Anaconda.
+ Accelerating Python* performance on modern architectures from Intel.
+"""
+
+# l_pythoni2_p_2019.2.066.tar.gz from the Intel site
+sources = ['l_pythoni%s_p_%s.tar.gz' % (pysver, relver)]
+checksums = ['dd0b73db5d0d79c3cdf779d10092eae32780d720b74ed8fde2a2c46e23143de1']
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+skipsteps = ['configure', 'build']
+
+buildininstalldir = True
+
+preinstallopts = """
+    echo -e "install:\n\t./setup_intel_python.sh" > Makefile
```

Comment:
@JackPerdue It looks like this could be handled via the `Binary` easyblock as well, by specifying `install_cmd`?